src/Input/InputEvents.cpp

```diff
@@ -479,3 +491,17 @@ InputEvents::ProcessTimer()
   DoQueuedEvents();
   ProcessMenuTimer();
 }
+
+void  //////// LOCKSCREEN
+InputEvents::eventLockscreen(const TCHAR *mode)
+{
+  LockScreen = !LockScreen;
+  if (LockScreen){
+    //Message::AddMessage("Lock Screen enabled");
+    ShowLockBox();
+    }else{
+    Message::AddMessage("Lock Screen disabled");
```

Comment:
Build failure

Reply:
I don't get this one...

Reply:
The usual `TCHAR` vs `char` conflict. On Windows, `TCHAR` maps to `wchar_t`.
A general question is: why is there even this message? What purpose does it serve?

Reply:
The message is to notify the user that the automatic lockscreen function is disabled

Reply:
That explains the purpose, but doesn't explain why you believe this is necessary. After all, the user just clicked the button, so he should know what he has just done.

Reply:
I would love to be able to change conditionally the test on the button, to have "enable lock screen" and "disable lock screen" but I am not, for the time being that was the better solution I could work out.